### ISSUE TITLE
Bump to hyper 0.14

### ISSUE THREAD

**Issue by mitchhentges** [routerify/routerify#34]
Projects using `tokio` 1.0 and `hyper` 0.14 can't use `routerify` yet :(

**Comment by seanpianka** [comment]
I've started updating the chain of dependencies here: https://github.com/rousan/async-pipe-rs/pull/8

The list of Routerify's dependencies that need to be updated are included in my pull request. We're just waiting on approvals for this work to proceed.

**Comment by seanpianka** [comment]
@rousan, could you invite me to the Routerify organization? I'd like to move this work along since the release of Tokio 1.0 is a major milestone that Routerify should keep up with.

I think it'd also be beneficial to move the stream-pipe and async-pipe dependencies into the organization, as well...

**Comment by seanpianka** [comment]
Here's a fork with Hyper and Tokio updated: https://github.com/seanpianka/routerify
Here's the pull request: #35 

To use the updated libraries, add this to your `Cargo.toml`:
```toml
routerify = { git = "https://github.com/seanpianka/routerify.git", branch = "master" }
routerify-cors = { git = "https://github.com/seanpianka/routerify-cors.git", branch = "master" }
routerify-query = { git = "https://github.com/seanpianka/routerify-query.git", branch = "master" }
```

**Comment by rousan** [comment]
@seanpianka , I have sent you an invitation link to be a member to this organization. Please follow this link https://github.com/routerify  to accept.

**Comment by gsserge** [comment]
Update is available in 2.0.0-beta-2.

**Comment by rousan** [comment]
Moved [async-pipe-rs](https://github.com/routerify/async-pipe-rs) and [stream-body ](https://github.com/routerify/stream-body) to this `routerify` organization from my personal repo list.
